.github/workflows/ci.yml

```
@@ -60,6 +60,9 @@ jobs:
           - name: '3.12'
             tox_env: integration-py312
 
+          - name: '3.13.0-alpha - 3.13.0'
```

Nit: You can do

```suggestion
          - name: ~3.13.0-0
```

to achieve the same.

I personally prefer this syntax because it allows me to conditionally allow failures if there's a tilde in the spec: https://github.com/cherrypy/cheroot/commit/99dff70d15e189b922420cd03688f5c039bbb49e#diff-662dff05c7dfe6681c1007ae601e8573d413a03f9f9d53d951c2cb99caba4fd4R704.

---

Another way that some people like is to pass `allow-prereleases: true` to `actions/setup-python`. It's been there since v4 IIRC because people wanted to just have `3.13` in their configs select pre-release versions.

---

Finally, it's also possible to have
```suggestion
          - name: 3.13-dev
```
as well.

---

Ah ha! I learned something new. Thanks!